### DISCUSSION TITLE
NextJS Turbo ReactStrictMode=false double console output in development

### DISCUSSION THREAD

**Discussion by DanielOberlechner** [vercel/turborepo#5176] [Help] [answered]
Hi,
I would like to ask, if there is a way to disable the double console.log output from turbo?
First I thought react strict mode is on, then I realised I only get the double outputs when the --turbo flag is enable in development.

Is there a way to disable it or is it needed wanted specifically ?

It distracts me when I'm @ developing because I think I have a loop or similar problem which causes this, so I would like to disable it. Checked already NextJS documentation and also Turbo's website but without any chances to disable it ....

Kind regards,
Daniel Oberlechner

**Comment by anthonyshew** [comment, marked as answer]
If you're still seeing this problem, please open an issue. 🙏
